repository/e.json

```
@@ -1,6 +1,18 @@
 {
 	"schema_version": "3.0.0",
 	"packages": [
+		{
+			"details": "https://github.com/kaessajidi/ekl-sublim-text",
+			"name": "Dassault Systemes EKL Syntax",
```

```suggestion
			"name": "Dassault Systemes EKL",
```

Language support packages don't need to have "syntax" or something like it in the name, since typically it does more than just the syntax: in your case it also provides completions (which you might want to specify in the labels). 

If you want to call it Dassault Systemes EKL and not just EKL, it should go into the d.json file instead, an in alphabetical order there.